doc/seg-schema.json

```diff
@@ -0,0 +1,135 @@
+      },
+
+      "SeriesDescription":
+      { "allOf" :
+        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/LO"},
+        {"properties" : {"default" : "Segmentation"}}]
+      },
+
+      "SeriesNumber":
+      { "allOf" :
+        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/IS"},
+        {"properties" : {"default" : "300"}}]
+      },
+
+      "InstanceNumber":
+      { "allOf" :
+        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/IS"},
+        {"properties" : {"default" : "1"}}]
+      },
+
+      "BodyPartExamined":
+      { "allOf" :
+        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/CS"},
+        {"properties" : {"default" : ""}}]
+      }
+    }
+  },
+
+  "segmentAttributes": {
+    "type" : "array",
+    "items" : {
+      "type" : "object",
+      "properties" : {
+        "LabelID": { "type" : "number", "default" : 1},
+        "SegmentDescription": { "type" : "string", "required" : false},
+
+        "SegmentedPropertyCategoryCodeSequence": {
+          "allOf" : [
+            {"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/codeSequence"},
+            {"properties" :
+             {"CodeValue" : {"default": "T-D0050"},
+              "CodingSchemeDesignator" : {"default": "SRT"},
+              "CodeMeaning" : {"default": "Tissue"}},
+            }
+          ],
+          "required" : true,
+        },
+
+        "SegmentedPropertyTypeCodeSequence": {
+          "allOf" : [
+            {"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/codeSequence"},
+            {"properties" :
+              {"CodeValue" : {"default": "T-D0050"},
+              "CodingSchemeDesignator" : {"default": "SRT"},
+              "CodeMeaning" : {"default": "Tissue"}},
+            }
+          ],
+          "required" : true,
+        },
+
+        "SegmentedPropertyTypeModifierCodeSequence" : {
+          "$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/codeSequence",
+          "required" : false
+        },
+
+        "AnatomicRegionSequence" : {
+          "$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/codeSequence",
+          "required" : false
+        },
+
+        "AnatomicRegionModifierSequence" : {
+          "$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/codeSequence",
+          "required" : false
+        },
+
+        "SegmentAlgorithmType": {
+          "allOf" : [
+            {"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/CS"},
+            {"enum" : ["AUTOMATIC","MANUAL","SEMIAUTOMATIC"]},
+            {"default" : "SEMIAUTOMATIC"}
+          ],
+          "required" : true,
+        },
+
+        "SegmentAlgorithmName": {
+          "$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/LO"
+        },
+
+        "recommendedDisplayRGBValue": {
```

Comment:
No, this is intentionally lowercase, since recommendedDisplayRGBValue is not a DICOM concept. CIELab is. I am following this convention to differentiate the two types of attributes.